src/app/services/postConfirmRemovePolicyCondition.js

```diff
@@ -0,0 +1,63 @@
+const {
+  getPolicyById,
+  updatePolicyById,
+} = require("../../infrastructure/access");
+const logger = require("../../infrastructure/logger");
+
+const postConfirmRemovePolicyCondition = async (req, res) => {
+  const condition = req.body.condition;
+  const operator = req.body.operator;
+  const value = req.body.value;
+
+  const policy = await getPolicyById(req.params.sid, req.params.pid, req.id);
+  const conditionOperatorAndValueInPolicy = policy.conditions.find(
+    (policyCondition) =>
+      policyCondition.field === condition &&
+      policyCondition.operator === operator &&
+      policyCondition.value.includes(value),
+  );
+
+  if (conditionOperatorAndValueInPolicy) {
+    if (conditionOperatorAndValueInPolicy.value.length === 1) {
+      // If only 1 value for condition, remove the whole condition from the policy
+      const conditionIndex = policy.conditions.indexOf(
+        conditionOperatorAndValueInPolicy,
+      );
+      policy.conditions.splice(conditionIndex, 1);
+    } else {
+      // If more than 1 value for condition, remove just that value from the array for that condition
+      const index = conditionOperatorAndValueInPolicy.value.indexOf(value);
+      conditionOperatorAndValueInPolicy.value.splice(index, 1);
+    }
+  } else {
+    // Very unlikely to ever get to this.  If someone tampered with the hidden values then it's possible.
+    logger.info(
+      `[${condition}] and [${operator}] and [${value}] not found in existing policy`,
+      { correlationId: req.id },
+    );
+  }
```

Comment:
Note so I don't forget.  Need to handle this properly.  As is, it works just fine but why make an API call when we could just not do anything instead! Much less risky.
Need to figure out what handling this 'properly' entails